src/angular/styles/_typography.scss

```diff
@@ -651,20 +651,20 @@
     &:where(.sbb-lean.sbb-dark) {
       --sbb-color-weak-accent: var(--sbb-color-milk);
 
-      // Checkbox
-      --sbb-checkbox-background-color-default: var(--sbb-color-background);
-      --sbb-checkbox-background-color-disabled: var(--sbb-color-charcoal);
+      // Checkbox & Radiobutton
+      --sbb-selection-container-background-color-default: var(--sbb-color-background);
+      --sbb-selection-container-background-color-disabled: var(--sbb-color-charcoal);
 
-      --sbb-checkbox-border-color-default: var(--sbb-color-graphite);
-      --sbb-checkbox-border-color-disabled: var(--sbb-color-smoke);
-      --sbb-checkbox-border-color-focused: var(--sbb-color-text);
+      --sbb-selection-container-border-color-default: var(--sbb-color-graphite);
+      --sbb-selection-container-border-color-disabled: var(--sbb-color-smoke);
+      --sbb-selection-container-border-color-focused: var(--sbb-color-text);
```

Comment:
This variable is currently not used.
It replace in `--sbb-checkbox-label-color-focused` in 
https://github.com/sbb-design-systems/sbb-angular/blob/03ef8f71af918fdd8831abd2066418271b7dc924/src/angular/styles/_typography.scss#L1905